psi4/__init__.py

```diff
@@ -98,3 +98,4 @@
 if "@ENABLE_PCMSolver@".upper() in ["1", "ON", "YES", "TRUE", "Y"]:
     sys.path.insert(1, "@PCMSolver_PYMOD@")
 
+from .broker import broker
```

Comment:
```
try:
    import ipi
    from .broker import broker
except ImportError:
    # Set some variable noting that this is not available
```

Not CMake required.

Reply:
I wanted to be able to import the broker below anyway so that it is possible to test parts of the infrastructure in the testsuite. Would it be possible to keep the `from .broker import broker` line due to that?

Reply:
Would it be possible to move broker into driver/ ? That way it’ll be installed automatically (as all .py files are) and you can add the from .broker import broker to psi4/driver/__init__.py

Reply:
Yes that works fine. Just pushed as well